### PR TITLE
Feat/import actions

### PR DESCRIPTION
**Pull request by Dominion5254**
* Import MyNode and RaspiBlitz actions
* Update upstream to 0.16.4-beta
* Remove Proxy as a dependency
* Expose Azeed CipherSeed in Properties for new wallets
* Fix non-deterministic bug causing Seed generation to fail on a fresh install
* Update dependencies.ts and autoconfig to accomodate pruned nodes contingent on Bitcoin Core version

### CODE REVIEW

File: scripts/services/migrations.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import { compat, matches, types as T } from "../deps.ts";
+import { literal } from "https://deno.land/x/ts_matches@v5.2.0/mod.ts";
```
Comment (elvece) [inline scripts/services/migrations.ts:2]: dont think you need this line

File: configurator/src/main.rs [hunk 2]
```diff
@@ -119,8 +119,8 @@ enum BitcoinCoreConfig {
     None,
     #[serde(rename_all = "kebab-case")]
     Internal { user: String, password: String },
-    #[serde(rename_all = "kebab-case")]
-    InternalProxy { user: String, password: String },
+    // #[serde(rename_all = "kebab-case")]
+    // InternalProxy { user: String, password: String },
```
Comment (elvece) [inline configurator/src/main.rs:123]: i would remove these comments if they are not needed, same with below

File: scripts/services/migrations.ts (hunk 1 above)
Comment (Dominion5254) [reply scripts/services/migrations.ts]: Removed

File: configurator/src/main.rs (hunk 2 above)
Comment (Dominion5254) [reply configurator/src/main.rs]: Removed

**Review by elvece — APPROVED** [review]

File: scripts/services/dependencies.ts [hunk 3]
```diff
@@ -1,177 +1,47 @@
...
       return { result: null };
     },
     // deno-lint-ignore require-await
     async autoConfigure(_effects, configInput) {
-      const config = matchBitcoindConfig.unsafeCast(configInput);
-      config["zmq-enabled"] = true;
-      return { result: config };
+      if (matchOldBitcoindConfig.test(configInput)) {
+        configInput.advanced.pruning.mode = "disabled"
+        configInput["zmq-enabled"] = true;
```
Comment (elvece) [inline scripts/services/dependencies.ts:39]: @dr-bonez did zmq go away in bitcoin 25?

Comment (elvece) [reply scripts/services/dependencies.ts:39]: nvm looks like it did

**Review by dr-bonez — APPROVED** [review]

**Pull request merged**